server/src/__tests__/executables.test.ts

```diff
@@ -1,44 +1,47 @@
 import * as path from 'path'
 import { Executables } from '../executables'
 
-const executables = Executables.fromPath(
+const executablesPromise = Executables.fromPath(
   path.resolve(__dirname, '..', '..', '..', 'testing', 'executables'),
 )
 
 describe('list', () => {
-  it('It finds executables on the PATH', () => {
+  it('finds executables on the PATH', async () => {
     expect.assertions(1)
```

Comment:
Not that you are using `await` you don't need:
- `expect.assertions(1)`
- and returning the last `expect`

Reply:
@mads-hartmann this was the reason why I suggested using `await` here, so we could get rid of these extra things you need to remember to do. : )